### PR TITLE
add handling for sell_loot yaml setting.

### PR DESCRIPTION
**Pull request by Hiinky**

### CODE REVIEW

File: locksmithing.lic [hunk 1]
```diff
@@ -77,7 +77,7 @@ class Locksmithing
       start_time = Time.now
       wait_for_script_to_complete('pick')
       if Time.now - start_time > 15
-        wait_for_script_to_complete('sell-loot')
+        wait_for_script_to_complete('sell-loot') if sell_loot
```
Comment (rpherbig) [inline locksmithing.lic:80]: ❓ Should this be `settings.sell_loot`? I don't see where `sell_loot` is defined

Comment (Hiinky) [reply locksmithing.lic]: Fixed!

**Comment by rpherbig** [comment]
@Hiinky I merged this before realizing you merged master into your branch. I think your merge was backwards. It should no longer be `settings.foo` and instead `@foo`. Can you look into this and make a new PR if needed?

**Comment by rpherbig** [comment]
Nevermind, I did some investigation. It looks like this was already addressed in #4265, so I reverted this change in #4292. `master` should be all good to go. Let me know if I messed something up.

**Pull request merged**